src/libs/humanizer/utils.ts

```diff
@@ -89,11 +88,8 @@ export async function getNativePrice(network: NetworkDescriptor, fetch: Function
     throw new Error(`getNativePrice: ${network.name} is not supported`)
   }
 
-  const baseUrl = COINGECKO_PRO_API_KEY
-    ? 'https://pro-api.coingecko.com/api/v3'
-    : 'https://api.coingecko.com/api/v3'
-  const postfix = COINGECKO_PRO_API_KEY ? `&x_cg_pro_api_key=${COINGECKO_PRO_API_KEY}` : ''
-  const coingeckoQueryUrl = `${baseUrl}/simple/price?ids=${platformId}&vs_currencies=usd${postfix}`
+  const baseUrl = 'https://cena.ambire.com/api/v3'
```

Comment:
Since this is used 2 times across this file, if you feel like this is needed - maybe we can move it on top of the files as `const BASE_URL = 'https://cena.ambire.com/api/v3'`?